### DISCUSSION TITLE
How dows it work in Windows?

### DISCUSSION THREAD

**Discussion by trusktr** [typicode/husky#1313] [Q&A]
Husky creates a commit hook with this in it:

```sh
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"

...
```

That works in Windows? Does Windows read the shebang and it has `sh`?

**Comment by typicode** [comment]
https://typicode.github.io/husky/troubleshooting.html#does-it-work-on-windows :wink:
